packages/client/package.json

```diff
@@ -5,14 +5,15 @@
   "description": "Experimental email client with social features",
   "author": "Jesse Hallett <jesse@sitr.us>",
   "license": "Apache-2.0",
+  "homepage": "./",
   "scripts": {
     "prestart": "cd ../common && yarn build",
-    "start": "cross-env BROWSER=none react-scripts start",
+    "start": "cross-env BROWSER=none react-app-rewired start",
     "prebuild": "cd ../common && yarn build",
-    "build": "react-scripts build",
+    "build": "cross-env INLINE_RUNTIME_CHUNK=false react-app-rewired build",
```

Comment:
Setting `INLINE_RUNTIME_CHUNK` configures create-react-app to use a non-inline script tag to load bootstrapping code. This is necessary so that we can put a nonce on that script tag.